src/hooks/useAsync.js

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+
+function useAsync(asyncFunction) {
+  const [pending, setPending] = useState(false);
```

Comment:
요런 boolean 형 변수일 때 다음과 같이 작성하곤 합니다

```suggestion
  const [isPending, setIsPending] = useState(false);
```